packages/tools/piw-utils-internal/src/widgets/Pagination.tsx

```diff
@@ -49,33 +60,38 @@ export function Pagination(props: PaginationProps): ReactElement | null {
                     props.previousPage();
                     setPageIndex(props.page - 1);
                 })}
+                aria-label={props.labelPreviousPage ?? "Go to previous page"}
             >
-                <span className="glyphicon glyphicon-backward" />
+                <span aria-hidden className="glyphicon glyphicon-backward" />
             </button>
-            <div className="paging-status">
-                {initialItem} to {lastItem}{" "}
-                {hasLastPage ? `of ${props.numberOfItems ?? (numberOfPages ?? 1) * props.pageSize}` : ""}
+            <span className="sr-only" tabIndex={0}>
+                {props.labelPagingStatus ?? "Currently showing"} {pagingStatus}
+            </span>
+            <div aria-hidden className="paging-status">
+                {pagingStatus}
```

Comment:
These couldn't be combined somehow?

Reply:
The first one is only available for Screen Readers and the other one is hidden for them. Also following same approach as Core Data Grid pagination.